core/src/main/java/org/apache/carbondata/core/datastore/chunk/store/impl/safe/AbstractNonDictionaryVectorFiller.java

```diff
@@ -48,7 +48,11 @@ public int getLengthFromBuffer(ByteBuffer buffer) {
   public static AbstractNonDictionaryVectorFiller getVectorFiller(DataType type, int lengthSize,
       int numberOfRows) {
     if (type == DataTypes.STRING) {
-      return new StringVectorFiller(lengthSize, numberOfRows);
+      if (lengthSize > 2) {
```

Comment:
2 is magic number, can you change to constant or add a function to make it more readable

Reply:
ok